modules/chimera/src/main/java/org/dcache/chimera/posix/UnixPermissionHandler.java

```diff
@@ -184,7 +184,7 @@ public boolean isAllowed(Acl acl, User user, int requsetedAcl) {
 
         } // switch( userUid )
 
-        _log.debug("IsAllowed: " + isAllowed);
+        logger.debug("IsAllowed: {}", isAllowed);
```

Comment:
@marisanest I suggest you close this pull request and create a new one, which includes this change only.